lib/ftu/signuplogin.dart

```diff
@@ -0,0 +1,253 @@
+//REF: https://dart.dev/guides/language/effective-dart/style#do-place-dart-imports-before-other-imports
+import 'package:flutter/material.dart';
+import '../constants/signuplogin_constants.dart';
+import './authentication.dart';
+
+
+// Changing from stateless to stateful
+class LoginSignupPage extends StatefulWidget {
```

Comment:
I think we should be separating the `Login` and `Signup` page into separate widgets, and reuse the appropriate UI elements.

Reply:
Great

Reply:
Peer programming needed for me

Reply:
Will make this in future PR.